shared/src/vyxal/Elements.scala

```diff
@@ -4049,6 +4049,31 @@ object Elements:
       false,
       "a: lst => indices of truthy elements of a",
     ) { a => ListHelpers.truthyIndices(ListHelpers.makeIterable(a)) },
+    addPart(
+      Monad,
+      "øA",
+      "Letter Number Swap",
+      List(
+        "letter-to-number",
+        "number-to-letter",
+        "letter-number-swap",
+        "number-letter-swap",
+        "a1-swap",
+      ),
+      true,
+      "a: str -> the index of a in the alphabet (one-indexed)",
+      "a: int -> the a-th letter of the alphabet (one-indexed)",
+    ) {
+      case a: VNum =>
+        "abcdefghijklmnopqrstuvwxyz" (math.floorMod((a - 1).toInt, 26)).toString
+      case a: String if a.length == 1 =>
+        VNum("abcdefghijklmnopqrstuvwxyz".indexOf(a.head.toLower) + 1)
+      case a: String => VList.from(
+          a.map(char =>
+            VNum("abcdefghijklmnopqrstuvwxyz".indexOf(char.toLower) + 1)
+          ).toList
+        )
```

Comment:
There's a teeny bit of code duplication here. Maybe try something like this?
```suggestion
      case a: String =>
        val inds =
          a.map(char =>
            VNum("abcdefghijklmnopqrstuvwxyz".indexOf(char.toLower) + 1)
          )
        if inds.length == 1 then inds.head else VList.from(inds)
```